applications/serial_lte_modem/doc/slm_data_mode.rst

```diff
@@ -49,16 +51,20 @@ Exiting data mode
 
 To exit data mode, the MCU sends the termination command set by the ``CONFIG_SLM_DATAMODE_TERMINATOR`` configuration option over UART and also applies the silence period set by the ``CONFIG_SLM_DATAMODE_SILENCE`` option, both before and after the termination command.
 
+When instructed to exit data mode, the SLM application returns the AT-command response ``OK``.
```

Comment:
```suggestion
When instructed to exit data mode, the SLM application returns the AT command response ``OK``.
```

Reply:
Please allow me to make the update in next PR.
I'm working on TCPIP proxy services that requires to update this document as well.

Reply:
Okay. Please fix this in the next one then.

Reply:
Yes, it will be. Thanks for the approval!